src/main/java/io/muserver/docs/samples/StreamingExample.java

```diff
@@ -0,0 +1,68 @@
+package io.muserver.docs.samples;
+
+import io.muserver.MuServer;
+import io.muserver.MuServerBuilder;
+import io.muserver.rest.RestHandlerBuilder;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.sql.*;
+
+public class StreamingExample {
+    public static void main(String[] args) {
+        FindOrders findOrders = new FindOrders();
+        MuServer server = MuServerBuilder.httpServer()
+            .addHandler(RestHandlerBuilder.restHandler(findOrders))
+            .start();
+        System.out.println("Started server at " + server.uri());
+    }
+
+    @Path("/stream/orders")
+    public static class FindOrders {
+        @GET
+        @Path("/all")
+        @Produces("application/json")
+        public Response get() {
+            String url = "jdbc:trino://localhost/schema/catalog?user=foo&password=bar";
+
+            StreamingOutput stream = new StreamingOutput() {
+                @Override
+                public void write(OutputStream os) throws IOException, WebApplicationException {
+                    try (
+                        Connection connection = DriverManager.getConnection(url);
+                        Statement statement = connection.createStatement();
+                        ResultSet rs = statement.executeQuery("SELECT orderkey, orderstatus FROM orders");
+                    ) {
+                        OutputStreamWriter writer = new OutputStreamWriter(os);
+                        writer.write("[\n");
+
+                        int count = 0;
+                        while (rs.next()) {
+                            if (count > 0) {
+                                writer.write(",\n");
+                            }
+                            String jsonR = new StringBuilder()
```

Comment:
It would be better to just to the writer directly. It is already buffered so no need to create a string builder.

I'm not a big fan of creating JSON this way as you need to json-encode everything properly... could use a org.json.JSONWriter but perhaps that is overcomplicated for this example, and just using strings is more straightforward for demo purposes. So okay.